api/openapi-spec/swagger.json

```diff
@@ -10765,6 +10765,10 @@
     "description": "Represents the stock pod network interface.",
     "type": "object",
     "properties": {
+     "vmIpv6NetworkCIDR": {
+      "description": "IPv6 CIDR for the vm network. Defaults to fd10:0:2::/120 if not specified.",
```

Comment:
The `if not specified` is redundant. `Defaults to` only has meaning when the parameter is not specified.

Not sure what's worse, being redundant or break consistency (the `vmNetworkCIDR` parameter features the same wording).

Reply:
As this is part of the API and potential documentation is generated from it, I prefer to stay safe and use the current format which mimics IPv4. But I do agree with you.
I will try to propose the text change in a separate PR to align both and see how it goes.

Reply:
Fair enough; thanks.